src/random_number.py

```diff
@@ -0,0 +1,8 @@
+from random import randint
+
+
+def main():
+    print("The random number is:", randint(1, 100))
+
+
+main()
```

Comment:
Vale lo stesso punto di review di su con https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do/